src/app/cases/page.tsx

```diff
@@ -76,29 +76,20 @@ export default function Page() {
         <FilterDropdown
           defaultValue="All countries"
           multi
-          // better solution available if we update ts target to es6
-          options={['All countries'].concat(
-            caseData
-              .map(c => c.country)
-              .filter((v, i, arr) => arr.indexOf(v) === i),
-          )}
+          options={[
+            'All countries',
+            ...Array.from(
```

Comment:
🤞 that the country of origin field is in a standardized format on LegalServer! also more generally, I suspect that the work of converting the LegalServer data into this form is going to be,,, curious,,, but we'll deal with that then

Reply:
🤞